contracts/VeYfiRewards.vy

```diff
@@ -43,7 +43,7 @@ struct Point:
     blk: uint256  # block
 
 
-TWO_WEEKS: constant(uint256) = 14 * 86400
+WEEK: constant(uint256) = 7 * 86400
```

Comment:
whats the intented design on lowering the rewards to 1 week instead of 2?

I believe it was an arbitrary time period, not 100% sure, but wondering whats the pro/con versus the prev setting.

Reply:
curve offers some details here
https://curve.readthedocs.io/dao-vecrv.html#implementation-details

Reply:
The idea here with two weeks on the original design was to distribute both gauges rewards and veYFI rewards every two weeks